release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -805,6 +805,54 @@ New Fluentd parameters allow you to performance-tune your Fluentd log collector.
 
 These parameters can help you determine the trade-offs between latency and throughput in your cluster logging instance.
 
+[id="ocp-4-6-monitoring"]
+=== Monitoring
+
+[id="ocp-4-6-monitoring-for-user-defined-projects"]
+==== Monitoring for user-defined projects
+
+In {product-title} 4.6, you can enable monitoring for user-defined projects in addition to the default platform monitoring. You can now monitor your own projects in {product-title} without the need for an additional monitoring solution. Using this new feature centralizes monitoring for core platform components and user-defined projects.
+
+With this new feature, you can perform the following tasks:
+
+* Enable and configure monitoring for user-defined projects
+* Create recording and alerting rules that use metrics from your own pods and services
+* Access metrics and information about alerts through a single, multi-tenant interface
+* Cross-correlate the metrics for user-defined projects with platform metrics
+
+For more information, see xref:../monitoring/understanding-the-monitoring-stack.adoc#understanding-the-monitoring-stack[Understanding the monitoring stack].
+
+[id="ocp-4-6-monitoring-alerting-rule-changes"]
+==== Alerting rule changes
+
+The following alerting rule changes have been introduced in {product-title} 4.6:
```

Comment:
```suggestion
The following alerting rule changes are introduced in {product-title} 4.6:
```
This is true for every instance of "has been" in this section - use the simplest tense you can.

Reply:
Thank you for highlighting this. Resolved.